node/validation/manifest/schema.ts

```diff
@@ -35,7 +34,7 @@ const functionConfigSchema = {
         type: 'string',
         format: 'regexPattern',
         errorMessage:
-          'excluded_patterns needs to be an array of regex that starts with ^ followed by / and ends with $ without any additional slashes before and afterwards',
+          'excluded_patterns must be an array of regex that starts with ^/ and ends with $ for example ^/blog/[d]{4}$',
```

Comment:
+1 to @eduardoboucas's suggestion here too. Though I do want to re-ask my open questions from https://github.com/netlify/edge-bundler/pull/339#discussion_r1140583208 because there are some discrepancies between this error message and [a related docs update](https://github.com/netlify/docs/pull/2943) and I'd like to figure out how to resolve that

1. Is `excluded_patterns` an internal syntax or is this the property that users and frameworks should be using? I ask because the [excluded patterns demo](https://github.com/netlify/excluded-patterns-demo) that I referenced while [documenting exclusions](https://github.com/netlify/docs/pull/2943) uses the syntax `excludedPattern` (camel case singular vs snake case plural)

2. Are customers/frameworks able to enter an array of regexes for excludedPattern or is the support for arrays here an internal implementation detail (maybe for handling multiple declarations for the same function)? If customers/frameworks can enter an array of regexes what is the syntax for doing that in `netlify.toml` and `manifest.json`? I ask because the [excluded patterns demo](https://github.com/netlify/excluded-patterns-demo) that I referenced while [documenting exclusions](https://github.com/netlify/docs/pull/2943) didn't indicate support for arrays. Here's a table of my understanding of what is supported for a single declaration in the various formats (this understanding is what my open docs PR is based on https://github.com/netlify/docs/pull/2943). Please correct anything that I've got wrong

|   | inline Config  | netlify.toml  | manifest.json  |
|---|---|---|---|
| path  | string or array of strings  | single string  |  single string |
| excludedPath |  string or array of strings |  single string |  single string |
| pattern  | not supported  | single regex  |  single regex |
| excludedPattern  | not supported | single regex  | single regex  |

Reply:
@eduardo should `excludePath` be allowed in `netlify.toml` or is there a reason it is not there?